libs/platform/src/lib/components/form/base.input.ts

```diff
@@ -37,12 +37,18 @@ let randomId = 0;
  * that e.g. placeholder will change after component is created
  */
 @Directive()
-export abstract class BaseInput extends BaseComponent
+export abstract class BaseInput
```

Comment:
all those changes in this file should be done in a separate PR where the documentation/formatting is addressed